cmd/collection/main.go

```diff
@@ -139,13 +141,25 @@ func main() {
 				"additional fraction of replica timeout that the primary will wait for votes")
 			flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
 				"the delay to broadcast block proposal in order to control block production rate")
+			flags.StringVar(&startupTimeString, "startup-time", cmd.NotSet,
+				"specifies date and time (in ISO 8601 format) when the consensus participant enters the first view (e.g 2006-01-02T15:04:05Z07:00)")
```

Comment:
I don't think we need this flag for collection cluster. 

Do we? @AlexHentschel

Reply:
The original issue says we need it for both.